pkg/apis/externaldns/types.go

```diff
@@ -45,6 +46,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	flags := pflag.NewFlagSet("", pflag.ContinueOnError)
 	flags.BoolVar(&cfg.InCluster, "in-cluster", false, "whether to use in-cluster config")
 	flags.StringVar(&cfg.KubeConfig, "kubeconfig", "", "path to a local kubeconfig file")
+	flags.StringVar(&cfg.Namespace, "namespace", "", "the namespace to look for endpoints")
```

Comment:
shouldn't this default to `v1.NamespaceAll`, i know this is empty string, but maybe make sense to be more explicit here and also mention, that if omitted it will track on all namespaces (as opposed do not track anything)